docs/src/changelog.md

```diff
@@ -50,6 +50,7 @@ See the [deployment instructions](https://quality-time.readthedocs.io/en/latest/
 
 - When using SonarQube security hotspots as source for the security warnings metric, allow for filtering hotspots by hotspot status ('to review', 'acknowledged', 'fixed', 'safe'). The default value of this parameter is to show hotspots 'to review' and 'acknowledged'. Closes [#5956](https://github.com/ICTU/quality-time/issues/5956).
 - Add axe analysis to Quality-time pipeline. Closes [#5402](https://github.com/ICTU/quality-time/issues/5402).
+- Add support for the SARIF format and `ruleId` lookup, in combination with the Violations metric. Closes [#6314](https://github.com/ICTU/quality-time/issues/6314).
```

Comment:
The `ruleId` is really an implementation detail of SARIF JSON files. Not sure it's necessary to mention it here. How about: 
```suggestion
- Add support for using the SARIF format as source for the 'violations' metric. This makes it possible to use [Robocop](https://robocop.readthedocs.io/en/stable/) as source for the violations metric. Closes [#6314](https://github.com/ICTU/quality-time/issues/6314).
```

Reply:
Nice yeah